tests/src/PatternAssertion/ContentBannerAssert.php

```diff
@@ -0,0 +1,191 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\Tests\oe_bootstrap_theme\PatternAssertion;
+
+use Symfony\Component\DomCrawler\Crawler;
+
+/**
+ * Assertions for the page header pattern.
+ */
+class ContentBannerAssert extends BasePatternAssert {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function getAssertions(string $variant): array {
+    return [
+      'image' => [
+        [$this, 'assertImage'],
+        '.card-img-top',
+      ],
+      'image_size' => [
+        [$this, 'assertImageSize'],
+      ],
+      'title' => [
+        [$this, 'assertElementText'],
+        '.card-title.bcl-heading',
+      ],
+      'title_tag' => [
+        [$this, 'assertTitleTag'],
+      ],
+      'background' => [
+        [$this, 'assertBackground'],
+      ],
+      'badges' => [
+        [$this, 'assertBadgesElements'],
```

Comment:
this `assertBadgesElements` is actually not defined, dangrous that it fails silently, if you have time you should look also into that, if not, make a separate ticket for it.

Same for ./tests/src/PatternAssertion/CardPatternAssert.php

Reply:
done. added in base pattern.